### ISSUE TITLE
Dshell Install Error in Arch Linux

### ISSUE THREAD

**Issue by dfirence** [USArmyResearchLab/Dshell#62]
Greetings, I am using Arch Linux and I wanted to install your framework.  Has anyone experienced the following error when running 'make'?

My Arch Version
## Linux ARCH-CND 4.0.7-2-ARCH #1 SMP PREEMPT Tue Jun 30 07:50:21 UTC 2015 x86_64 GNU/Linux

My Python Version
## My Error Seen

IPy is installed
pcap is installed
Crypto is installed
dpkt is installed
# Generating .dshellrc and dshell files

python /usr/share/Dshell/bin/generate-dshellrc.py /usr/share/Dshell
  File "/usr/share/Dshell/bin/generate-dshellrc.py", line 28
    except Exception, e:
                    ^
SyntaxError: invalid syntax
Makefile:8: recipe for target 'rc' failed
make: **\* [rc] Error 1

**Comment by dev195** [comment]
You mention that you call python as "Python2".
Inside the Makefile, a call to just "python" is made to run the generate-dshellrc.py script.
You may need to go into the Makefile briefly to switch that from "python" to "Python2", if you haven't already.

**Comment by dfirence** [comment]
Hi,

I think I got things to work now.  In my Arch environment I run Python3 and Python2.  So what I found the issue is, is the way your pre-packages bundle would expect the Arch environment to map everything like it would be in Ubuntu.  I draw this because I modified your 'MakeFile' as you suggested and I modified the 'env' of your python scripts that get dropped in the < dshell/bin > directory.
----------------------------------------------------------------------------------------------------------------------------

Will let you know how it goes. Thank you for your contribution to the community.
